index.html

```diff
@@ -36,20 +36,20 @@ <h1>I’m Abhishek
     <div class="Links">
     <ul>
        
-       <li><img src="assets/images/Vector.svg" alt="icon1"></li>
+       <li><img src="assets/images/Vector.svg" alt="This is My Twitter account page"></li>
        
-       <li><img src="assets/images/Linkedin.svg" alt="icon2"></li>
-       <li><img src="assets/images/Vector (1).svg" alt="icon3"></li>
-       <li><img src="assets/images/Vector (2).svg" alt="icon4"></li>
-       <li><img src="assets/images/Vector (3).svg" alt="icon5"></li>
+       <li><img src="assets/images/Linkedin.svg" alt="This is My Linkedin account page"></li>
+       <li><img src="assets/images/Vector (1).svg" alt="This is My Medium account page"></li>
+       <li><img src="assets/images/Vector (2).svg" alt="This is My Github account page"></li>
+       <li><img src="assets/images/Vector (3).svg" alt="This is My account page"></li>
```

Comment:
- [ ] Kindly make sure to change your alternative text. The alternative text does not need to include the words "button", "link", or "image of". (Screen readers automatically provide that information.) :wink:  Please make sure to change every alternative text.
   
  - ### Suggestion
     Change  **_This is My Linkedin account page_** to _**Linkedin**_